packages/Card/src/components/Footer/Footer.styles.js

```diff
@@ -5,10 +5,12 @@ import tokens from "@paprika/tokens";
 export const footerStyles = styled.div`
   align-items: center;
   background: ${tokens.color.blackLighten70};
+  border-bottom: 1px solid ${tokens.color.blackLighten60};
   border-radius: 0 0 6px 6px;
   box-sizing: border-box;
   color: ${tokens.color.black};
   display: flex;
+  justify-content: space-between;
   ${stylers.fontSize(-1)};
   min-height: ${stylers.spacer(6)};
   padding-left: ${stylers.spacer(2)};
```

Comment:
I think this needs to have padding on the right too.  This shouldn't be allowed to happen:
![Screen Shot 2021-12-07 at 1 27 02 PM](https://user-images.githubusercontent.com/14944896/145108743-5e815563-c8d6-41cf-832e-c744712cd040.png)